conf/application.conf

```diff
@@ -211,17 +211,15 @@ services-config {
     friendlyName = "the_New_Computerised_Transit_System"
     friendlyNameWelsh = "y_System_Gludo_Gyfrifiadurol_Newydd"
   }
-  ars {
 
-  enrolment = "HMRC-ATAR-ORG"
-  shortName = "ars"
-  callBack = "http://localhost:9582/advance-tariff-application",
-  accessibility = "http://localhost:12346/accessibility-statement/advance-valuation-ruling",
-  friendlyName = "advance_rulings"
-  friendlyNameWelsh = "ddyfarniadau_uwch"
+  ars {
+    enrolment = "HMRC-ATAR-ORG"
+    shortName = "ARS"
+    callBack = "http://localhost:12600/advance-valuation-ruling",
+    accessibility = "http://localhost:12346/accessibility-statement/advance-valuation-ruling",
+    friendlyName = "Advance_Valuation_Ruling"
+    friendlyNameWelsh = "ddyfarniadau_uwch"
```

Comment:
I would double check the friendlyName as we were specifically ased to use advance_rulings by a member of your team. (Also, we'd need the welsh updated)

Reply:
https://github.com/hmrc/eori-common-component-frontend/pull/438/commits/e7635ab84a2c6e4fb17a8cce2301d643351fa691